unitconvert.py

```diff
@@ -19,17 +19,102 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
 # FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 # DEALINGS IN THE SOFTWARE.
-
 """Unit conversions"""
 
-from time import mktime
-from datetime import datetime
+import datetime
+ZERO = datetime.timedelta(0)
+
 
 def semicircle_to_degrees(semicircles):
     """Convert a number in semicricles to degrees"""
     return semicircles * (180.0 / 2.0 ** 31)
 
+
 def local_date_to_utc(date):
     """Local date to UTC"""
-    return datetime.utcfromtimestamp(mktime(date.timetuple()))
+    return date.replace(tzinfo=UTC)
+
+
+class UTC(datetime.tzinfo):
```

Comment:
:thinking: 

Why not just use the built-in [`datetime.timezone.utc`](https://docs.python.org/3/library/datetime.html#datetime.timezone.utc)?

```py
from datetime import timezone
UTC = timezone.utc
```

Reply:
I believe I was using python 2.7 at the time.
Should I fix this? Seven years with no activity sound like no one is particularly interested.

Reply:
Yeah, sorry, I noticed that after reviewing this. :man_facepalming: :joy:

BTW, I looked into this myself, and _all_ of the timestamps used in `fittotcx` (lap start, activity ID, and GPS timestamp) are already in UTC… so I wonder how an offset can get in there.